photon-core/src/main/java/org/photonvision/vision/pipeline/AdvancedPipelineSettings.java

```diff
@@ -90,6 +92,60 @@ public AdvancedPipelineSettings() {
     public int cornerDetectionSideCount = 4;
     public double cornerDetectionAccuracyPercentage = 10;
 
+    // Static cropping settings
+    public int static_x = 0;
+    public int static_y = 0;
+    public int static_width = Integer.MAX_VALUE;
+    public int static_height = Integer.MAX_VALUE;
+
+    private final NTDataChangeListener dynamicCropListener;
+    private IntegerArraySubscriber dynamicCropRequest;
+    private final IntegerArrayPublisher ledModeState;
+
+    public int dynamic_x = 0;
+    public int dynamic_y = 0;
+    public int dynamic_width = Integer.MAX_VALUE;
+    public int dynamic_height = Integer.MAX_VALUE;
```

Comment:
Does this make sense to serialize?

More in general I'm not sure how I feel about dynamic crop via NT -- it's pretty perpendicular to all our other stuff to date.